src/Components/LoginOut.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import LoginForm from "./LoginForm";
+import JournalistCards from "./JournalistCards";
+import Parse from "parse";
+
+function LoginOut(props) {
+  process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
+
+  const adminUser = {
+    email: "admin@admin.com",
+    password: "admin123",
+  };
```

Comment:
Guessing this is temporary for testing?
There are better ways to do this. I don't expect you to know the answer right now, just know that there are better ways :)